k8s/Makefile

```diff
@@ -35,6 +35,7 @@ export APP_EMAIL_HOST ?= email-smtp.us-west-2.amazonaws.com
 export APP_EMAIL_PORT ?= 587
 export APP_EMAIL_USE_TLS ?= True
 export APP_SECURE_HSTS_SECONDS = 31536000
+export APP_AUTOMATICALLY_INGEST_CONTENT ?= True
```

Comment:
Heya @limed could you confirm i have all the pieces of the puzzle to get `settings.AUTOMATICALLY_INGEST_CONTENT` set up as `True` in all environments, please?

Reply:
You will need to expose this in `app.env.yaml.j2` file as well

Reply:
> You will need to expose this in `app.env.yaml.j2` file as well

Didn't scroll down enough, looks like you did do that so you should be good :D